scripts/log-session.py

```
@@ -0,0 +1,334 @@
+    lines.extend(['', '---', ''])
+
+    return '\n'.join(lines)
+
+
+def emit_event(entry: SessionEntry) -> None:
+    """Emit event to events.jsonl for FS5 consumption (GAP-5)."""
+    memory_dir = get_memory_dir()
+    events_file = memory_dir / 'events.jsonl'
+
+    event = {
+        "timestamp": entry.timestamp.astimezone(timezone.utc).isoformat(),
```

Ruff (0.14.14)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)